README.md

```diff
@@ -44,17 +44,18 @@ all isolates are paused before collecting coverage.
 #### Formatting coverage data
 
 ```
-pub global run coverage:format_coverage --packages=app_package/.packages -i coverage.json
+pub global run coverage:format_coverage --package=app_package -i coverage.json
```

Comment:
To pile on: should we update this to `dart pub global...`

Reply:
Done